src/models/methods/ledger.ts

```diff
@@ -0,0 +1,56 @@
+import { LedgerEntry } from "../ledger";
+import { BaseRequest, BaseResponse } from "./baseMethod";
+
+export interface LedgerRequest extends BaseRequest {
+    command: "ledger"
+    ledger_hash?: string,
+    ledger_index?: string | number
+    full?: boolean
+    accounts?: boolean
+    transactions?: boolean
+    expand?: boolean
+    owner_funds?: boolean
+    binary?: boolean
+    queue?: boolean
+}
+
+interface Ledger {
+    account_hash: string
+    accountState?: LedgerEntry[]
+    close_flags: number
+    close_time: number
+    close_time_human: string
+    close_time_resolution: number
+    closed: boolean
+    ledger_hash: string
+    ledger_index: string
+    parent_close_time: number
+    parent_hash: string
+    total_coins: string
+    transaction_hash: string 
+    transactions?: (string | JSON)[] //https://xrpl.org/ledger.html
```

Comment:
You can leave this as `any` and leave a `TODO` to retype this when we have actual transaction types